packages/picasso-lab/src/Section/Section.tsx

```diff
@@ -136,8 +136,9 @@ export const Section = forwardRef<HTMLDivElement, Props>(function Section(
       {hasHeader && (
         <Container
           data-testid={testIds?.header}
-          className={cx(classes.header, {
-            [classes.collapsedHeader]: collapsed
+          className={cx({
+            [classes[`header${capitalize(variant)}`]]: true,
+            [classes[`collapsedHeader${capitalize(variant)}`]]: collapsed
```

Comment:
wdyt, what if we place `variant` on the first place, so we may avoid usage of `capitalize`:
```suggestion
            [classes[`${variant}Header`]]: true,
            [classes[`${variant}CollapsedHeader`]]: collapsed
```